rpc/subscription.go

```diff
@@ -78,7 +79,7 @@ func NotifierFromContext(ctx context.Context) (*Notifier, bool) {
 // are dropped until the subscription is marked as active. This is done
 // by the RPC server after the subscription ID is send to the client.
 func (n *Notifier) CreateSubscription() *Subscription {
-	s := &Subscription{NewID(), make(chan error)}
+	s := &Subscription{ID: NewID(), err: make(chan error)}
```

Comment:
The problem is that the subscription callback calls the `CreateSubscription` and has no clue under which namespace it is exposed. The RPC server knows it and sets it when the subscription is activated.